content/en/platform/corda/1.6/cenm/release-notes.md

```diff
@@ -14,6 +14,23 @@ title: Release notes
 
 # Corda Enterprise Network Manager release notes
 
+## Corda Enterprise Network Manager 1.6.1
+
+### New features and enhancements
+
+* `sshdHost` can now be specified within the `shell {}` config block for the CENM services that support Shell.
+* Added a new `host` option in the `enmListener` configuration.
+* Added a new `host` option in the `adminListener` configuration.
+* The Zone Service can now be started either with a configuration file or with command-line options. Any existing deployments using the original method remain unaffected.
+* The Angel Service can now be started either with a configuration file or with command-line options. Any existing deployments using the original method remain unaffected.
```

Comment:
... can now be started either with a configuration file or with the existing command-line options. Any current deployments using the original method remain unaffected.

(for both angel and zone) I think on reflection this sounds a bit better

Reply:
updated